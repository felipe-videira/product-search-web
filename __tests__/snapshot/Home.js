import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../pages';

describe('Home', () => {
  it('Should render without crashing', () => {
    const tree = renderer.create(<Home />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
