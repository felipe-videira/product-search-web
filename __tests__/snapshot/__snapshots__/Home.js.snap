// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render home page 1`] = `
<div>
  <nav>
    <img
      alt="Mercado Libre"
      height="50"
      src="/assets/images/logo.png"
      width="75"
    />
    <form
      action="/items"
      method="GET"
      name="searchBar"
      onSubmit={[Function]}
    >
      <input
        name="search"
        placeholder="Nunca dejes de buscar"
        type="text"
      />
      <button
        type="submit"
      >
        <img
          alt="Busca"
          height="20"
          src="/assets/images/search.svg"
          width="20"
        />
      </button>
    </form>
  </nav>
</div>
`;
